src/wovnio/wovnphp/Utils.php

```diff
@@ -8,7 +8,10 @@ class Utils
     // will return the store and headers objects
     public static function getStoreAndHeaders(&$env)
     {
-        $file = isset($env['WOVN_CONFIG']) ? $env['WOVN_CONFIG'] : DIRNAME(__FILE__) . '/../../../../wovn.ini';
+        $file = isset($env['WOVN_CONFIG']) ? $env['WOVN_CONFIG'] : DIRNAME(__FILE__) . '/../../../../wovn.json';
+        if (!file_exists($file)) {
```

Comment:
How about using `WOVN_CONFIG`, when user needs to use json format.
Then, WOVN.php doesn't need to check wovn.json exist or not.

Reply:
I don't quite get this one. Can you explain it a bit more?

Reply:
When client set like `$_SERVER['WOVN_CONFIG'] = '/path/to/wovn.json'`, WOVN.php will see wovn.json.
And we don't need to change here.
json user is miner now, we don't need to check wovn.json every time.